doors_dashboards/dashboards/dashboard.py

```diff
@@ -97,6 +98,8 @@ def create_dashboard(config: Dict) -> html.Div:
             dcc.Store(id=store_ids["collection_selector"]),
             dcc.Store(id=store_ids["group_selector"]),
             dcc.Store(id=store_ids["variable_selector"]),
+            dcc.Interval(id=f"{dashboard_id}-interval", interval=1 * 1000,
```

Comment:
Can you make this a parameter in the configuration? Then we can set it to daily for some configurations and to a much larger period for others, where we don't expect updates. It would be even better if in those cases we wouldn't even have an Interval component.